src/Builder/ListBuilder.php

```diff
@@ -129,25 +129,25 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
                 switch ($fieldDescription->getMappingType()) {
                     case ClassMetadataInfo::MANY_TO_ONE:
                         $fieldDescription->setTemplate(
-                            'SonataDoctrineORMAdminBundle:CRUD:list_orm_many_to_one.html.twig'
+                            'SonataAdminBundle:CRUD/Association:list_many_to_one.html.twig'
```

Comment:
No change in the `composer.json` for this?

Reply:
@core23 I added todo for this, waiting for a new release of SonataAdmin.